src/tokens/saga.ts

```diff
@@ -197,22 +207,17 @@ export function* fetchTokenBalancesSaga() {
     }
     SentryTransactionHub.startTransaction(SentryTransaction.fetch_balances)
 
-    const importedTokens = (yield* call(
+    const showImportedTokens = yield* call(
       getFeatureGate,
       StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW
-    ))
-      ? yield* select(importedTokensInfoSelector)
-      : {}
-    const supportedTokens = yield* call(getTokensInfo)
+    )
+    const importedTokens = yield* select(importedTokensInfoSelector)
```

Comment:
i wonder if this selector should just return `[]` if the feature flag is off. that would remove the need for every place that consumes imported tokens to also check the flag, and would remove the need to check this flag later in this function (calling `fetchImportedTokenBalances` with an empty array seems okay?)

Reply:
Is it ok to fetch the ff inside a selector??
The ff doesn't change without an app restart, right?
So I guess yes, it's fine.